src/web/components/KeyPairs/KeyPairsTable.tsx

```diff
@@ -59,20 +64,28 @@ function KeyPairsTable({
       <table className='key-pairs-table'>
         <thead>
           <tr>
-            <th className='name'>Name</th>
-            <th className='subscription-id'>Subscription ID</th>
+            <SortableTableHeader<KeyPairModel> className='name' sortKey='name' header='Name' />
+            <SortableTableHeader<KeyPairModel>
+              className='subscription-id'
+              sortKey='subscriptionId'
+              header='Subscription ID'
+            />
             <th className='public-key'>Public Key</th>
-            <th className='created'>Created</th>
+            <SortableTableHeader<KeyPairModel>
+              className='created'
+              sortKey='created'
+              header='Created'
+            />
             <th className='action'>Actions</th>
           </tr>
         </thead>
         <tbody>
-          {keyPairs &&
-            keyPairs.map((k) => (
+          {sortedKeyPairs &&
```

Comment:
Since we are defaulting to an empty array on line 40, I think we can get rid of this check. So I think we can just have:
```
<tbody>
  {sortedKeyPairs.map((k) => (
    <KeyPair
      key={k.publicKey}
      keyPair={k}
      existingKeyPairs={sortedKeyPairs}
      onEdit={onKeyPairEdit}
      onDisable={onKeyPairDisable}
    />
  ))}
</tbody>
```

Reply:
sonarlint suggests using an optional chain but I don't think we even need that.